### ISSUE TITLE
bug: users cannot reopen issues

### ISSUE THREAD

**Issue by Airblader** [i3/i3-github-bot#4]
See i3/#1953. If the bot closes the issue, it says

```
[…] verify the bug still exists, and re-open this issue.
```

But normal users cannot reopen issues.

**Comment by stapelberg** [comment]
Missed this issue because I was accidentally not watching this repository.

I think the fix might simply be to reword the message. What do you think about this:

> […] verify the bug still exists, and we will re-open the issue once you comment on it again.

**Comment by nmschulte** [comment]
bugs are not automatically closed anymore -- this issue seems out-dated as the response is no longer misleading

**Comment by Airblader** [comment]
Sorry, I don't understand. Users still cannot reopen issues on their own, AFAIK, or has this changed? The message definitely hasn't.

**Comment by nmschulte** [comment]
users still cannot re-open their issue, true, but the bot no longer misleads (at least not me):
![](https://cloud.githubusercontent.com/assets/8540239/24787319/49c3ac6a-1b2d-11e7-98e3-9537cc72f8da.png)

Perhaps the bot closes long-standing issues and I never see those messages?

**Comment by stapelberg** [comment]
The bot (still) auto-closes messages filed for versions which are not the most recent major version.

**Comment by Airblader** [comment]
Yes, the bot makes a difference between no version being given and the given version not being the most recent major version.
